src/go/monitor/main.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+func main() {
+	http.Handle("/metrics", promhttp.Handler())
```

Comment:
Can we extract this server into a separate struct and add a test?

Reply:
I added a simple test using `httptest`